### PR TITLE
Replaced old requestAnimFrame with new polyfill

### PR DESCRIPTION
**Pull request by jaredwilli**
This is potentially going to be the version you should use to polyfill requestAnimationFrame, and it has an added cancelAnimationFrame also which could be useful maybe.

Paul has even updated his blog post to recommend using this one instead.

### CODE REVIEW

File: boxbox.js [hunk 1]
```diff
@@ -15,18 +15,32 @@ Created at Bocoup http://bocoup.com
...
-               window.setTimeout(callback, 1000 / 60);
-           };
-}());
+// Erik Moller's requestAnimationFrame shim
+// http://my.opera.com/emoller/blog/2011/12/20/requestanimationframe-for-smart-er-animating
+(function() {
+    var lastTime = 0;
+    var vendors = ['ms', 'moz', 'webkit', 'o'];
+    for(var x = 0; x < vendors.length && !window.requestAnimationFrame; ++x) {
+        window.requestAnimationFrame = window[vendors[x]+'RequestAnimationFrame'];
```
Comment (jaredwilli) [inline boxbox.js:24]: Might want to make this requestAnimFrame.
and other places it defines it as window.requestAnimationFrame/cancelAnimationFrame...

Comment (jaredwilli) [inline boxbox.js:24]: I also just saw this which you may want to check out as well, as it relates to requestAnimFrame implementations
http://updates.html5rocks.com/2012/05/requestAnimationFrame-API-now-with-sub-millisecond-precision

**Comment by incompl** [comment]
Thanks for the PR, it just needed the actual code to use requestAnimationFrame instead of requestAnimFrame then it worked like a charm.

**Comment by jaredwilli** [comment]
Cool :)

**Pull request merged**